### PR TITLE
Fixes #637, search button fixed at all dimensions(big and small)

### LINKED ISSUE #637: At smaller screen size, search button is not working.
**Actual Behaviour**

Clicking on the search button icon does not show results.

**Expected Behaviour**

It should show results when search icon has been clicked.

**Screenshots of the issue**

screenshot from 2017-07-12 18-44-14

Reference issue: #651

### PR DESCRIPTION
**Pull request by Marauderer97**
Fixes issue #637 

Changes: search button fixed at all dimensions(big and small), I also made some changes in html and css because the audio and search button had the same id which might cause problems in the future.

Demo Link: [https://marauderer97.github.io/susper.com/](https://marauderer97.github.io/susper.com/)

Screenshots for the change: Not applicable here

[Add here the screenshot of the fix]

### CODE REVIEW

File: src/app/search-bar/search-bar.component.ts [hunk 1]
```diff
@@ -75,6 +75,20 @@ export class SearchBarComponent implements OnInit, AfterViewInit {
...
+  onClick() {
+    if (this.searchdata.fq !== '') {
+      this.store.dispatch(new queryactions.QueryServerAction({'query': this.searchdata.query, start: 0, rows: this.searchdata.rows, fq: this.searchdata.fq}));
+    } else {
+      this.store.dispatch(new queryactions.QueryServerAction({
+        'query': this.searchdata.query,
+        start: 0,
+        rows: this.searchdata.rows
+      }));
+    }
```
Comment (harshit98) [inline src/app/search-bar/search-bar.component.ts]: Have you indented this code? Just asking, to make sure there are no new any changes inside `onClick()`

**Comment by harshit98** [comment]
Referencing this PR to #651 too.

File: src/app/search-bar/search-bar.component.ts (hunk 1 above)
Comment (harshit98) [inline src/app/search-bar/search-bar.component.ts]: Also, @Marauderer97 fix Travis issues which are occurring.

Comment (Marauderer97) [inline src/app/search-bar/search-bar.component.ts]: @harshit98 yes indented and cross checked with ng lint, travis trying to figure out

**Comment by Marauderer97** [comment]
@harshit98 @nikhilrayaprolu I removed one test case which says that the search bar should always have the search button, this was failing because there is no search button in the index page, and since the speech button had the same id, it used to pass all this time. Now that I gave them both seperate ids it was failing

**Comment by nikhilrayaprolu** [comment]
@Marauderer97 one should not use the same id for two elements. please remove that.

**Comment by Marauderer97** [comment]
@nikhilrayaprolu I don't understand, I just did

**Pull request merged**